cabal.project

```diff
@@ -2,11 +2,11 @@ packages: pandoc.cabal
 
 package pandoc
   flags: +embed_data_files -trypandoc
-  ghc-options: -Wall -fno-warn-unused-do-bind -Wincomplete-record-updates -Wnoncanonical-monad-instances -Wnoncanonical-monadfail-instances -Wincomplete-uni-patterns -Widentities -j +RTS -A32M -RTS
+  ghc-options: -fhide-source-paths -j +RTS -A32M -RTS
 
 package pandoc-citeproc
   flags: +embed_data_files +bibutils -unicode_collation -test_citeproc -debug
-  ghc-options: -Wall -fno-warn-unused-do-bind -Wincomplete-record-updates -Wnoncanonical-monad-instances -Wnoncanonical-monadfail-instances -Wincomplete-uni-patterns -Widentities -j +RTS -A32M -RTS
+  ghc-options: -fhide-source-paths -j +RTS -A32M -RTS
```

Comment:
Are you sure that options here get added to the ghc-options in the cabal file, rather than replacing them altogether?
I think I believed the latter, perhaps as a result of experience.
But this should be easy to test.

Reply:
Options were combined when I tested it (cabal-install 2.4.0.0).

Reply:
OK, great.

Reply:
And if we put `-fhide-source-paths` in pandoc-citeproc.cabal, we're just left with `-j +RTS -A32M -RTS`.  I wonder if those should also go in pandoc-citeproc.cabal?